hyperspy/defaults_parser.py

```diff
@@ -138,11 +138,16 @@ class GUIs(t.HasTraits):
 
 
 class PlotConfig(t.HasTraits):
-    saturated_pixels = t.CFloat(0.05,
```

Comment:
I think that we shouldn't break the API for this release.

* I suggest to deprecate ``saturated_pixels`` instead for removal in 2.0.
* Change its default value to 0 (it will only affect new users with no ``hyperspyrc`` file.

I think that it was a poor idea to have this in preferences since there is no default value that is always good, so I think that it is better to remove it for good.